src/lib/browser/Browser.ts

```diff
@@ -1,12 +1,18 @@
-import puppeteer from 'puppeteer-core';
-import type { Browser as BrowserInterface } from 'puppeteer-core/lib/esm/puppeteer/api-docs-entry';
+import type {
+  Browser as BrowserInterface,
+  BrowserContext,
+  BrowserContextOptions,
+} from 'playwright';
+import { chromium } from 'playwright';
```

Comment:
```suggestion
import { chromium } from 'playwright-chromium';
```

sounds like you are only interested in chromium, so I'd change it to that and then `npx playwright install` will also only install chromium.

Reply:
ah I wasn't aware of that, thanks!